### PR TITLE
Fix some issues found by pylint

### PR DESCRIPTION
**Pull request by jmcs**

### CODE REVIEW

File: senza/__init__.py [hunk 1]
```diff
@@ -1 +1,6 @@
+"""
+Deploy immutable application stacks and create and execute AWS CloudFormation
+templates in a sane way Edit
```
Comment (rafaelcaricio) [inline senza/__init__.py]: " Edit" ?

File: senza/traffic.py [hunk 2]
```diff
@@ -401,7 +420,9 @@ def inform_sns(arns: list, message: str, region):
     sns_topics = set(arns)
     sns = BotoClientProxy('sns', region_name=region)
     for sns_topic in sns_topics:
-        sns.publish(TopicArn=sns_topic, Subject="SenzaTrafficRedirect", Message=jsonizer.encode((message)))
+        sns.publish(TopicArn=sns_topic,
+                    Subject="SenzaTrafficRedirect",
+                    Message=jsonizer.encode((message)))
```
Comment (rafaelcaricio) [inline senza/traffic.py]: Too many `()`

File: senza/__init__.py (hunk 1 above)
Comment (jmcs) [inline senza/__init__.py]: Fixed

File: senza/traffic.py (hunk 2 above)
Comment (jmcs) [inline senza/traffic.py]: Also fixed

**Comment by rafaelcaricio** [comment]
👍

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**